routes/web.php

```diff
@@ -22,6 +22,8 @@
 Route::get('campaigns/{id}/actions/create', 'ActionsController@create');
 Route::resource('campaigns', 'CampaignsController', ['except' => ['index', 'show']]);
 Route::resource('group-types', 'GroupTypesController', ['except' => ['index', 'show']]);
+Route::get('group-types/{id}/groups/create', 'GroupsController@create');
+Route::resource('groups', 'GroupsController', ['except' => 'show']);
```

Comment:
Do we want to `except` the standalone `create` route as well, or are we cool with creating Groups without a group type ID? (It looks like the create action expects a `$groupTypeId`)

Reply:
Navigating to `/groups/create` will fail the same way navigating to `/actions/create` will fail:
```
Type error: Too few arguments to function Rogue\Http\Controllers\Web\GroupsController::create(), 0 passed and exactly 1 expected
```
We are not cool with creating Groups without a Group Type ID -- but this does prevent it from happening. This is also following by example from the ActionsController, where we don't want to create an Action without a Campaign.

Reply:
Ah, gotcha. Wouldn't it make sense than to explicitly exclude it from the resource since it's not a route we intend to support, instead of just letting it break? Either way, certainly not a blocker here, and I understand that this is a follow-by-example.

Reply:
Speaking of `gotcha` -- if we do exclude `create`, we'll get the React rendered route `/groups/{id}`, with an error of 
```
Error: GraphQL error: Variable "$id" of non-null type "Int!" must not be null.
```
All this nitty gritty feels like good examples of why we have the outstanding TODO to move these Laravel web controllers into React components GraphQL migrations:
https://github.com/DoSomething/rogue/blob/f660e391e988b84aad379b11066c3d3c9138a366/routes/web.php#L19-L22

But we decided to go this route for sake of keeping dev in a single repo (and hoping to get most groups backend work done this sprint). refs https://github.com/DoSomething/rogue/pull/1026#discussion_r432152226

Reply:
Oy, _gotcha_ -- looking forward to that migration!